test/helpers/vertical_nav_helper_test.rb

```diff
@@ -26,7 +26,7 @@ def setup
 
       # if not permitted
       stubs(can?: false)
-      assert_equal(["Backend Overview", "Methods and Metrics", "Mapping Rules"], backend_api_nav_sections.pluck(:title))
+      assert_equal(["Backend Overview"], backend_api_nav_sections.pluck(:title))
```

Comment:
This test is a bit silly, stubbing the whole `can?` method, but probably testing all different combinations could be an overkill too.